src/App.css

```diff
@@ -1,3 +1,6 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;600;800&display=swap");
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;600;800&family=Roboto+Slab:wght@200;400;600&display=swap");
+
 .App {
```

Comment:
- [ ] Make sure styling is EXACTLY the same as the one provided on [Zeplin](https://zeplin.io/). Let's implement the changes requested by the previous reviewer to stick to the same design. Kindly, compare yours vs Zeplin design below 👇 

- Your design 👇 
![student_LI](https://user-images.githubusercontent.com/67087664/146563964-2b743e39-bb55-4636-a3d6-1052dd922ebc.jpg)
- Zeplin design 👇 
![zeplin_LI](https://user-images.githubusercontent.com/67087664/146564030-91019eb2-b8e0-410b-8fe3-7c756427e52e.jpg)